Tests/scripts/download_demisto_conf.sh

```diff
@@ -45,9 +45,11 @@ if [ "$?" != "0" ]; then
     fi
     rm -rf ./content-test-conf-$UNDERSCORE_CIRCLE_BRANCH
     rm -rf ./test_configuration.zip
+    echo "ERROR: Found a branch with the same name in contest-test-conf conf.json - $UNDERSCORE_CIRCLE_BRANCH.\n Merge it in order to merge the current branch into content repo."
+    exit 1
+
 fi
 
 set -e
-echo "using instance:"
-cat instance.json
+rm instance.json
```

Comment:
remove this line

Reply:
Why? We said we don't want it

Reply:
we are not using the instance so the print message shouldn't be there.
but no need to delete it.
instead, you can remove lines 39-45